src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -128,6 +128,27 @@ function containsOtherElements(element) {
 	return element.children.length > 0;
 }
 
+function getElementXPath(node) {
```

Comment:
We really need tests for js&mdash;sooner than later!

Reply:
Totally in for that! :-)